examples/gno.land/p/demo/memeland/memeland_test.gno

```diff
@@ -97,7 +98,7 @@ func TestGetPostsInRangeByTimestamp(t *testing.T) {
 
 	// Count the number of posts returned in the JSON string as a rudimentary check for correct pagination/filtering
 	postCount := strings.Count(jsonStr, `"id":"`)
-	if postCount != m.MemeCounter {
+	if seqid.ID(postCount) != m.MemeCounter {
```

Comment:
Why change this & line #161 exactly?

Reply:
hey, that's because the type of  postCount is int, and the base type of MemeCounter is uint64, IIRC,  so the types are mismatch and not comparable.
the modification is needed to do an explicit conversion to make them comparable.

this is actually a bug indemnified by the gno type check.